Source/CarthageKit/Xcode.swift

```diff
@@ -1154,9 +1193,9 @@ private func binaryURL(_ packageURL: URL) -> Result<URL, CarthageError> {
 
 /// Signs a framework with the given codesigning identity.
 private func codesign(_ frameworkURL: URL, _ expandedIdentity: String) -> SignalProducer<(), CarthageError> {
-	let codesignTask = Task("/usr/bin/xcrun", arguments: [ "codesign", "--force", "--sign", expandedIdentity, "--preserve-metadata=identifier,entitlements", frameworkURL.path ])
-
-	return codesignTask.launch()
-		.mapError(CarthageError.taskError)
-		.then(SignalProducer<(), CarthageError>.empty)
+	let codesignTask = Task(
+		"/usr/bin/xcrun",
+		arguments: ["codesign", "--force", "--sign", expandedIdentity, "--preserve-metadata=identifier,entitlements", frameworkURL.path]
+	)
+	return codesignTask.launch().mapError(CarthageError.taskError).then(SignalProducer<(), CarthageError>.empty)
```

Comment:
All reactive operators in most cases. Simple single-line expressions are okay IMO.